src/components/header/Header.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <v-container>
+    <v-row class="-centtexter">
+      <v-col class="mb-4">
+        <h1 class="title-todo">
+          todos
+        </h1>
+      </v-col>
+    </v-row>
+    <TodoFooter />
+  </v-container>
+</template>
+
+<script>
+  import TodoFooter from '@/components/todo-footer/TodoFooter.vue'
+
+  export default {
+    name: 'Header',
+
+    components: {
+      TodoFooter
+    },
+
+    data: () => ({})
```

Comment:
This doesn't contain any property (state) that we are using in the component, please remove it